File: FreeMarketClient/Interface.cpp
#include "Interface.h"
#include <cstdio>
namespace FreeMarket
{
    PlayerConfig::PlayerConfig()
        : id()
        , team()
    { }
        
    GameConfig::GameConfig()
        : turns()
        , turnTimeLimit()
        , setupTimeLimit()
        , player()
    { }
        
    Player::Player()
        : id()
        , team()
        , money()
    { }
        
    Game::Game()
        : turn()
        , player()
        , activeAuction()
        , endedAuction()
        , futureAuction()
    { }
        
}

File: FreeMarketClientCs/Interface.cs
using System;

namespace FreeMarket
{

    public class PlayerConfig
    {
        public int Id;
        public int Team;
        
    }

    public class GameConfig
    {
        public int Turns;
        public int TurnTimeLimit;
        public int SetupTimeLimit;
        public PlayerConfig[] Player;
        
    }

    public class PlayerState
    {
        public int Id;
        public int Team;
        public int Money;
        
    }

    public class GameState
    {
        public int Turn;
        public PlayerState[] Player;
        public Auction[] ActiveAuction;
        public Auction[] EndedAuction;
        public Auction[] FutureAuction;
        
    }
}